kubetest/e2e.go

```diff
@@ -229,7 +229,7 @@ func run(deploy deployer, o options) error {
 
 	var kubemarkWg sync.WaitGroup
 	var kubemarkDownErr error
-	if o.kubemark {
+	if o.down && o.kubemark {
```

Comment:
Just to confirm, all our CI tests call kubetest with --down, right?
So this change would be no-op for them, but will allow us to disable kubemark down in manual tests?

Reply:
I think so -- without this, we wouldn't tear down the main cluster after the test and I think that we do that.

I also checked few random build logs and all calls kubetest with --down.

Reply:
there are soak tests that doesn't tear down the cluster, but that won't be affected.